pylint/extensions/docparams.py

```diff
@@ -83,6 +83,12 @@ class DocstringParameterChecker(BaseChecker):
                   # we can't use the same old_name for two different warnings
                   # {'old_names': [('W9009', 'missing-yields-doc')]},
                  ),
+        'W9015': ('"%s" differing in parameter documentation',
```

Comment:
In order to be consistent with the old disables, we need to add ``old_names`` to these two news errors. ``old_names`` means that if the user disabled ``missing-param-doc``, it will have the new ``differing-param-doc`` disabled as well.

```'old_names': [('numeric-id', 'symbolic-id')]}```